custom/icds_reports/static/js/directives/prevalence-of-severe-report/prevalence-of-severe-report.directive.js

```diff
@@ -3,10 +3,11 @@
 var url = hqImport('hqwebapp/js/initial_page_data').reverse;
 
 function PrevalenceOfSevereReportController($scope, $routeParams, $location, $filter, maternalChildService,
-    locationsService, userLocationId, storageService, genders, ages, haveAccessToAllLocations,
-    baseControllersService, haveAccessToFeatures, isAlertActive) {
+    locationsService, dateHelperService, navigationService, userLocationId, storageService, genders, ages,
+    haveAccessToAllLocations, baseControllersService, haveAccessToFeatures, isAlertActive) {
     baseControllersService.BaseController.call(this, $scope, $routeParams, $location, locationsService,
-        userLocationId, storageService, haveAccessToAllLocations, haveAccessToFeatures);
+        dateHelperService, navigationService, userLocationId, storageService, haveAccessToAllLocations,
+        haveAccessToFeatures);
```

Comment:
this is the first of about 15-20 commits that apply the same treatment to every subclass of `BaseController` (injecting the `dateHelperService` and `navigationService`. I did each one in a single commit so that I could track that I had tested everything on web one by one. I also did a very thorough grepping exercise after I was done to ensure that I caught every reference to `BaseController`. I'll leave it up to the reviewer how much you want to trust me versus review the next 15 or so commits carefully.